data/hai/hai culture conversations.txt

```diff
@@ -207,3 +207,57 @@ mission "Gambling Judgment"
+			`	A friendly Hai emerges to take your order, and politely informs you that there's a 10% surcharge for eating on the terrace, but that's hardly a meaningful sum to a ship's captain so you just nod and try and work out what you're eating. Eventually you settle for a sandwich with acorn butter and vegetable slices that appear to be spiced, or pickled, you're not really sure. It was the staff's top recommended item and didn't come with any follow-up questions about Hai food preferences that you couldn't answer.`
+				goto finishingchat
+			label pleasantlunch
+			`	The interface is rather annoying to work with human-sized hands, and eventually you settle for a sandwich with acorn butter and vegetable slices that appear to be spiced, or pickled, you're not really sure. It was the top recommended item and didn't come with any follow-up questions about Hai food preferences that you couldn't answer.`
+			label finishingchat
+			`	As you eat your strange, but actually surprisingly tasty, lunch you take note of the text written large above the gateway into the grounds for the building complex opposite you. After a moment your translator box takes note of where you're looking and reads it aloud for you.`
+			`	'To know is to lead, to study is to know, to serve is to study, to lead is to serve.'`
+			`	Naturally, your translator cannot answer questions, so you eat your lunch and leave it be until a staff member emerges to take your plate.`
+			choice	
+				`	(Ask them about it.)`
+				`	(Keep your thoughts to yourself.)`
+					goto question
+			`	The staff member takes your plate and wishes you a lovely day.`
+				decline
+			label question
+			`	As they clear your plate from the table you ask them about the text on the gateway.`
+			`	"Oh, that's the city Academy. It's a... uh..." she pauses for a moment and relays something in Hai which your translator immediately converts to, "non-technical university."`
+			`	Giving the box a surprised look she smiles and continues in Hai, "If you want to be a manager, a politician, a social worker, or even just a public speaker, you go to the Academy and do courses on leadership and communications. Strictly speaking, it's not necessary to go, but lots of places simply won't accept you into those sorts of roles without some sort of Academy qualification, and they really do provide some great skills. I'm actually going there next year to do a proper human language course."`
+			`	"Technically the Academies are all independent, but they all share the same motto. It's an obligatory thing too though. That's what the motto is about. I'm told most humans think it's backwards, but it's not. The Academies keep tabs on their graduates; if you lead with self-interest, serve without studying, study without learning or waste your knowledge without applying it, they've been known to revoke people's qualifications. Attendance is free, so there's an expectation that you don't waste what they spend time teaching you.`
```

Comment:
Maybe this could help tone down on the periods?
```suggestion
			`	"Technically the Academies are all independent, but they all share the same motto. It's an obligatory thing too though, that's what the motto is about. I'm told most humans think it's backwards, but it's not. The Academies keep tabs on their graduates; if you lead with self-interest, serve without studying, study without learning or waste your knowledge without applying it, they've been known to revoke people's qualifications. Attendance is free, so there's an expectation that you don't waste what they spend time teaching you.`
```

Reply:
This in itself is a pretty big improvement.